File: multiples-of-3-and-5/main.cpp
#include <iostream>
#include <vector>

int main() {
    const int n = 1000;

    int total = 0;

    for (int i = 0; i < n; ++i) {
        if (i % 3 == 0 || i % 5 == 0) {
            total += i;
        }
    }

    std::cout << total;
}

File: multiples-of-3-and-5/main.go
package main

import "fmt"

func main() {
	var n, total int = 1000, 0

	for i := 0; i < n; i++ {
		if i%3 == 0 || i%5 == 0 {
			total += i
		}
	}
	fmt.Println(total)
}
